libs/wal_decoder/proto/interpreted_wal.proto

```diff
@@ -5,6 +5,7 @@ package interpreted_wal;
 message InterpretedWalRecords {
   repeated InterpretedWalRecord records = 1;
   optional uint64 next_record_lsn = 2;
```

Comment:
I think it should be safe to remove the `optional` here, if all callers are guaranteed to always set it. They kind of have to, since we `expect()` this now.

Reply:
Counter-intuitively, it's safer to keep it as optional. If the field is optional and None is provided, it doesn't get serialized. Unwrap fails on the pagserver side and we learn about it. If it's not optional, the default value of 0 gets used. I guess you could check for 0 on the PS side too.

Either way, I don't think it makes a difference.

Reply:
Nah, it's just a bit annoying to have this `Option` that can never be `None`.